tests/rpc/utils/utils.test.ts

```diff
@@ -14,29 +12,6 @@ describe('utils tests', () => {
     },
     { numRuns: 1000 },
   );
-
-  testProp(
-    'can get the head message',
-    [rpcTestUtils.jsonMessagesArb],
-    async (messages) => {
-      const { firstMessageProm, headTransformStream } =
-        rpcUtils.extractFirstMessageTransform(rpcUtils.parseJSONRPCRequest);
-      const parsedStream = rpcTestUtils
-        .messagesToReadableStream(messages)
-        .pipeThrough(rpcTestUtils.binaryStreamToSnippedStream([7]))
-        .pipeThrough(headTransformStream)
-        .pipeThrough(
-          middleware.binaryToJsonMessageStream(rpcUtils.parseJSONRPCMessage),
-        ); // Converting back.
-
-      expect(await firstMessageProm).toStrictEqual(messages[0]);
-      expect(await AsyncIterable.as(parsedStream).toArray()).toStrictEqual(
-        messages.slice(1),
-      );
-    },
-    { numRuns: 1000 },
-  );
-
   // TODO:
   //  - Test for badly structured data
```

Comment:
Arbitrary todos here. All todos should be tickable tasks on the PR!

Reply:
These should be issues.

Reply:
Fixed.